packages/api/src/external/carequality/command/cq-directory/get-cq-directory-entries.ts

```diff
@@ -0,0 +1,66 @@
+import { QueryTypes } from "sequelize";
+import { Pagination, sortForPagination } from "../../../../command/pagination";
+import { CQDirectoryEntry2 } from "../../cq-directory";
+import { HIEDirectoryEntryViewModel } from "../../models/hie-directory-view";
+import { paginationSqlExpressions } from "../../../../shared/sql";
+
+export async function getHieDirectoryEntriesByFilter(
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  { filter, pagination }: { filter: string; pagination: Pagination }
+): Promise<CQDirectoryEntry2[]> {
+  const sequelize = HIEDirectoryEntryViewModel.sequelize;
+  if (!sequelize) throw new Error("Sequelize not found");
+
+  const querySelect = `SELECT * FROM ${HIEDirectoryEntryViewModel.tableName} WHERE 1 = 1 `;
```

Comment:
I also saw this `tableName` this week as I was working on CQ dir stuff. We have a property on models called `NAME` with the table name.

Initially, I thought: "let's get rid of this and just use Sequelize's prop. The issue is that we might be using the code in the context where we didn't initialize the model, just Sequelize's connection pool (like in utils), so then I assume this prop won't be accessible. So I ended up using the `NAME` we define manually (and use to indicate the table name to Sequelize).

Reply:
Ahh interesting, good call. Makes a lot of sense, thanks for the heads up here.